cli/ingress-controller/flags.go

```diff
@@ -53,9 +53,10 @@ type cliConfig struct {
 	KongAdminCACertPath    string
 
 	// Resource filtering
-	WatchNamespace string
-	IngressClass   string
-	ElectionID     string
+	WatchNamespace      string
+	SkipEmptyAnnotation bool
```

Comment:
Please rename this field to `SkipClasslessIngressV1beta1`. Please rename other fields accordingly.